docs/Release Notes.md

```diff
@@ -65,25 +65,20 @@ The Ansible Modules for Dell PowerScale support the following features:
 - Get details, modify, import, and delete SyncIQ certificates.
 - Get details and modify SMB global settings.
 - Get details and modify SNMP settings.
+- Import, modify, setting default, and delete a server certificate.
   
 The Ansible modules use playbooks, written in yaml syntax, to list, show, create, delete, and modify each of these entities.
 
 New Features and Enhancements
 ---------------------------
 This section describes the features of the Ansible Modules for Dell PowerScale for this release.
 
-The Ansible Modules for Dell PowerScale release 2.4.1 supports the following features:
+The Ansible Modules for Dell PowerScale release 2.5.0 supports the following features:
 
-- The SMB global settings module supports this functionality:
-    - Added support for getting and modifying SMB global settings.
-- The SNMP settings module supports this functionality:
-    - Added support for getting and modifying SNMP settings.
-- The network pool module supports this functionality:
-    - Added support for removing the static route for IP address pool.
-- The settings module has been enhanced to support this functionality:
-    - Added support for getting and modifying cluster owner information and cluster identity information.
+- The server certificate module supports this functionality:
+    - Added support for importing, modifying, setting defaults, and deleting server certificates.
 - The Info module has been enhanced to support this functionality.
-    - Added support for listing SMB global settings, detailed network interfaces, NTP servers, email settings, cluster identity, cluster owner and SNMP settings through info module.
+    - Added support for listing server certificates.
 
 Known issues
 ------------
```

Comment:
Update version below in this file: https://github.com/dell/ansible-powerscale/pull/72/files#diff-dbb84812da420b1d22115c2206447d6f70be5e618d263ed43d3a2518f0bbe692R127

Reply:
Yes, updated the 2.4.1 --> 2.5.0 version